config/environments/development.rb

```diff
@@ -1,3 +1,5 @@
+include ApplicationHelper
```

Comment:
We definitely shouldn't be including any custom helpers in the environment files; in any case, the use below is comparing a value, not a boolean, so we shouldn't be using the helper here anyway.